### PR TITLE
tidy up dependencies

### PR DESCRIPTION
**Pull request by elharo**
@hboutemy Looks like this doesn't link the resolver API directly after all

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -87,17 +87,21 @@
       <version>${mavenVersion}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.maven</groupId>
+      <artifactId>maven-model</artifactId>
+      <version>${mavenVersion}</version>
```
Comment (hboutemy) [inline pom.xml:93]: provided scope, please

File: pom.xml [hunk 2]
```diff
@@ -87,17 +87,21 @@
...
       <scope>provided</scope>
     </dependency>
     <dependency>
-      <groupId>org.apache.maven.resolver</groupId>
-      <artifactId>maven-resolver-api</artifactId>
-      <version>1.4.1</version>
-      <scope>provided</scope>
+      <groupId>org.eclipse.aether</groupId>
+      <artifactId>aether-api</artifactId>
+      <version>0.9.0.M2</version>
```
Comment (hboutemy) [inline pom.xml:104]: provided scope also, please

File: pom.xml (hunk 1 above)
Comment (elharo) [reply pom.xml:93]: done

File: pom.xml (hunk 2 above)
Comment (elharo) [reply pom.xml:105]: done

**Pull request merged**